src/complex-list-field/complex-list-field.component.html

```diff
@@ -42,8 +42,14 @@
         <tr *ngFor="let key of keys[i] | addAlwaysShowFields:schema.items | filterAndSortBySchema:schema.items; trackBy:trackByFunction">
           <td class="label-holder">
             <div>
-              <label>{{key | underscoreToSpace}}</label>
-              <add-new-element-button *ngIf="schema.items.properties[key].type === 'array'" [path]="getValuePath(pIndex, key)" [schema]="schema.items.properties[key]"></add-new-element-button>
+              <title-dropdown [title]="key | underscoreToSpace">
```

Comment:
Can you make the style of this dropdown like the bootstrap one? (See that the whole row is clickable and has a background on hover)

![dropdown](https://cloud.githubusercontent.com/assets/580727/21018982/4454325c-bd6f-11e6-8473-32dfe88ef02b.gif)